prog/vm/nexus.rb

```diff
@@ -305,6 +305,11 @@ def before_run
   end
 
   label def destroy
+    when_prevent_destroy_set? do
+      puts "#{vm} Destroy prevented by the semaphore, decrease it to continue"
+      nap 30
```

Comment:
Maybe add a deadline for 1 day (or more), so that we wouldn't forget about this.

Reply:
I added it